huaweicloud/services/vpn/resource_huaweicloud_vpn_gateway.go

```diff
@@ -242,6 +249,97 @@ func ResourceGateway() *schema.Resource {
 	}
 }
 
+func GatewayCertificateSchema() *schema.Resource {
```

Comment:
it's unnecessary to export this function.
```suggestion
func gatewayCertificateSchema() *schema.Resource {
```